AEPAnalytics/Tests/FunctionalTests/AnalyticsTrack+LifecycleTestBase.swift

```diff
@@ -15,6 +15,7 @@ import AEPServices
 @testable import AEPAnalytics
 @testable import AEPCore
 
+@available(tvOSApplicationExtension, unavailable)
```

Comment:
It looks like when adding tvOS support it implicitly enables tvOSApplicationExtension. We need to explicitly annotate unavailable else it throws error, since `Analytics` class is not availale for `tvOSApplicationExtension`